keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -141,8 +139,8 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
 
       test("retrieving config by application returns an empty list") {
```

Comment:
```suggestion
      test("retrieving config by application fails") {
```

Reply:
why should it fail? maybe an empty list is better user experience?

Reply:
ok I just saw that it supposed to fail and return an exception, so nevermind!